src/utils/mediaQueries.ts

```diff
@@ -1,25 +1,33 @@
 import { Command } from 'types'
+import { media } from 'theme/styles/media'
 
 export type Breakpoints = 'xl' | 'lg' | 'md' | 'sm' | 'xs'
 
+const extrapolatedScreenSize = {
```

Comment:
@alongoni I have taken as `media screen sizes` for the `Breakpoints` the same ones used in the `styles`, this should solve what was driving you crazy.